lib/warden/strategies/session_strategy.rb

```diff
@@ -4,7 +4,11 @@ def valid?
   end
 
   def authenticate!
-    redirect!('/login', origin: request.path)
+    if request.content_type == 'application/json'
```

Comment:
I don't. I'm wondering if I still need this, since we want you to be able to use session auth to access the api. 

Lemme think on this after I address the other comments.